src/screens/SettingsScreen.tsx

```diff
@@ -0,0 +1,277 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Text, TouchableOpacity, Image, Switch } from 'react-native';
+import { Signs, SettingScreenTitles, SettingScreenContentText, SettingScreenTheme } from '@utils/constants';
+import colors from '@styles/colors';
+import { MAIN_HORIZONTAL_OFFSET } from '@styles/constants';
+
+export const SettingsScreen: React.FC = () => {
+    const [typeCelsius, setTypeCelsius] = useState(true);
+
+    const [isEnabled, setIsEnabled] = useState(false);
+
+    const toggleSwitch = () => setIsEnabled(previousState => !previousState);
+
+    const [systemTheme, setSystemTheme] = useState(true);
+    const [blackTheme, setBlackTheme] = useState(true);
+
+    const changeType = (isCelisius: boolean) => {
+        typeCelsius !== isCelisius && setTypeCelsius(isCelisius);
+    };
+
+    const changeTheme = (isSystem: boolean, isBlack: boolean) => {
```

Comment:
I don't understand what this function does. Also you should get system color value somewhere